jaggeryapps/admin/serviceproviders/custom/js/editsp.js

```diff
@@ -597,7 +597,7 @@ function updateCustomSP(file) {
                data: formData
     })
         .done(function (data) {
-            if (data) {
+            if (data && data.trim()!= "") {
```

Comment:
1. Need to explain why this seemingly hack needed.
2. data.trim() != "", please note a space between trim() and !=